src/IndexGame.py

```diff
@@ -81,6 +92,7 @@ def displayGameMenu(turn, map, selectedBuilding):
     #display building remaining
     #display map
     #display 2 randomly selected building via selectedBuilding
+    randomBuilding()
```

Comment:
As mentioned, put the run function in the Main program instead of the display menu. 

The display menu is in charge of collecting the user input and displaying the options to the user.